packages/@guardian/source-foundations/src/typography/font-styles.ts

```diff
@@ -103,37 +106,54 @@ export const fontStyleFunction =
 		};
 	};
 
+/**
+ * Helper method that returns a valid CSS string for a given set of typography styles.
+ */
+const fontStyleToCssString = (styles: TypographyStyles) => {
+	const { fontFamily, fontSize, fontStyle, fontWeight, lineHeight } = styles;
+
+	return `
+			font-family: ${fontFamily};
+			font-size: ${typeof fontSize === 'number' ? `${fontSize}px` : fontSize};
+			line-height: ${lineHeight};
+			${fontWeight ? `font-weight: ${fontWeight};` : ''}
+			${fontStyle ? `font-style: ${fontStyle};` : ''}
+		`;
```

Comment:
This adds tabs to the string and makes it multi-line. Maybe simply an array that’s joined with return carriages?

```ts
return [
	`font-family: ${fontFamily};`,
	`font-size: ${typeof fontSize === 'number' ? `${fontSize}px` : fontSize};`,
	`line-height: ${lineHeight};`,
	fontWeight && `font-weight: ${fontWeight};`,
	fontStyle && `font-style: ${fontStyle};`,
].filter(Boolean).join("\n");
```

Reply:
If we change this I feel like we'd need to change all examples of CSS defined as template strings, as this is a common pattern throughout this package (see: https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/utils/resets.ts#L64) — my feeling is that by switching to an array in this one case we might be introducing something that could surprise any future developers working on source-foundations.

There might also be a discussion to be had around readability vs the benefits of shipping the CSS without tabs